.prettierignore

```diff
@@ -1,6 +1,7 @@
 storybook-static
 node_modules
 lib
+src/tokens-dist
```

Comment:
Added src/tokens-dist because otherwise prettier tries to fix some stuff up, and the linting steps fail on CI.

Figured it makes sense to just go with the formatting style-dictionary gives us for these.